### PR TITLE
research/report-from-work-done-on-signature-verification-issue.md: in…

### PR DESCRIPTION
**Pull request by rg3mdeb**
…itial commit

Signed-off-by: Robert Goniszewski <robert.goniszewski@3mdeb.com>

### CODE REVIEW

File: research/report-from-work-done-on-signature-verification-issue.md [hunk 1]
```diff
@@ -0,0 +1,58 @@
+# Report From Work Done On Signature Verification Issue
+
+> Problem is described within [Vitro Oracle PoC Release Notes](https://github.com/VitroTech/release-notes/blob/2fb0391512e492c5d875828b1b4b90d98557e8b4/2020/oracle-poc-2020-07-22/release-notes.md).
+
+### Steps Done
+
+ 1. Research on core concepts related to the issue. As it's a more complex
```
Comment (artur-rs) [inline research/report-from-work-done-on-signature-verification-issue.md:7]: @rg3mdeb maybe we shouldn't write timeframes on each task + do not write in the first-person in the docs/reports

File: research/report-from-work-done-on-signature-verification-issue.md [hunk 2]
```diff
@@ -0,0 +1,58 @@
...
+  	payloadSigned = append(payloadSigned, s.Bytes()...)
+  	fmt.Println("signature takes first:", len(payloadSigned), " bytes of payload")
+  	payloadSigned = append(payloadSigned, []byte(payload)...)
+  	fmt.Println("total payload size:", len(payloadSigned))
+    ```
+
+
+### Conclusion
+
+As there was no additional possible solutions in sight, and due to limited time designated for this task, there was no further attempts made. Documentation is scarce on this matter, so it's hard to find an description on how to attach the keys to the payload.
```
Comment (artur-rs) [inline research/report-from-work-done-on-signature-verification-issue.md]: @rg3mdeb wrap to 80

File: research/report-from-work-done-on-signature-verification-issue.md (hunk 1 above)
Comment (rg3mdeb) [reply research/report-from-work-done-on-signature-verification-issue.md:7]: Won't happen again. Fixed

File: research/report-from-work-done-on-signature-verification-issue.md (hunk 2 above)
Comment (rg3mdeb) [reply research/report-from-work-done-on-signature-verification-issue.md]: Fixed

**Review by artur-rs — APPROVED** [review]

**Pull request merged**